bokehjs/src/lib/core/util/arrayable.ts

```diff
@@ -35,7 +38,20 @@ export function copy<T>(array: Arrayable<T>): Arrayable<T> {
   }
 }
 
+export function splice<T>(array: T[], start: number, k?: number, ...items: T[]): T[]
+export function splice<T>(array: Arrayable<T>, start: number, k?: number, ...items: T[]): Arrayable<T>
+
 export function splice<T>(array: Arrayable<T>, start: number, k?: number, ...items: T[]): Arrayable<T> {
```

Comment:
I would maybe have named `k` `count`, but that is not related to this PR.

Reply:
It should be called `delete_count` to match `splice()`'s original signature.